packages/module/src/CodeModal/CodeModal.tsx

```diff
@@ -126,6 +140,11 @@ export const CodeModal: React.FunctionComponent<CodeModalProps> = ({
       </ModalFooter>
     </Modal>
   );
+
+  if (displayMode === ChatbotDisplayMode.fullscreen && isModalOpen) {
+    return <div className="pf-v6-c-backdrop pf-chatbot__backdrop">{modal}</div>;
+  }
```

Comment:
The existing modal backdrop doesn't show above the chatbot due to the way the chatbot is positioned. We also can't pass classes to it. This lets us fake it for now.